app/javascript/src/app/controllers/submit_on_enter_controller.js

```diff
@@ -0,0 +1,15 @@
+import { Controller } from 'stimulus';
+import moment from 'moment';
+
+export default class extends Controller {
+  connect() {
+    jQuery(this.element).keydown(function(event) {
+      const keypressed = event.keyCode || event.which;
+      if (keypressed == 13) {
+        jQuery(this.element)
+          .closest('form')
+          .submit();
+      }
+    });
+  }
+}
```

Comment:
I'd prefer we do this without `jQuery`. Also `moment` doesn't seem to be required.